client/src/pages/Social.jsx

```diff
@@ -87,44 +102,84 @@ class Social extends Component {
       .catch(err => console.log(err));
     console.log(this.props);
   };
+
   render() {
+    const { classes } = this.props;
     return (
       <div>
-        User Posts Page Components Workspace:
-        <br />
-        <CheckIn />
-        <Post
-          handleInputChange={this.handleInputChange}
-          handleFormSubmit={this.handleFormSubmit}
-        />
-        <Paper className={styles.paper} elevation={20}>
-          <Grid container wrap="nowrap" spacing={16}>
-            <Grid item xs>
-              <Typography variant="h5" component="h3">
-                Current Location Updates:
-              </Typography>
-              <hr />
-              <Paper className={styles.list} elevation={20}>
-                <List>
-                  {this.state.otherPosts.map(post => {
-                    return (
-                      <ListItem key={post._id}>
-                        {post.userName}
-                        <br />
-                        {post.date}
-                        <hr />
-                        {post.post}
-                      </ListItem>
-                    );
-                  })}
-                </List>
-              </Paper>
+        {withStyles}
+        <Paper className={classes.paper} elevation={20}>
+          <CheckIn />
+          <Post
+            handleInputChange={this.handleInputChange}
+            handleFormSubmit={this.handleFormSubmit}
+          />
+          <br />
+          <Paper className={classes.posts} elevation={20}>
+            <Grid container wrap="nowrap" spacing={16}>
+              <Grid item xs>
+                <Typography variant="h5" component="h3">
+                  Current Location Updates:
+                </Typography>
+                <hr />
+                <Paper className={classes.list} elevation={20}>
+                  <List className={classes.postStyle} id="list">
+                    {this.state.otherPosts.map(post => {
+                      return (
+                        <Typography>
+                          <ListItem key={post._id} alignItems="flex-start">
+                            <ListItemAvatar>
+                              <Avatar
+                                src={this.state.currentUser.picture}
```

Comment:
I think that there's a flaw in how this is being added to what's pulled from the database. Not all posts that are displayed in the posts area will be the current user that's logged in. Instead, we'd want to send the user nickname and avatar along with the data that's getting saved to the 'posts' collection in the 'handleFormSubmit' function above.